console/console-init/ui/src/modules/address-space/components/AddressSpaceList/AddressSpaceList.tsx

```diff
@@ -201,19 +231,60 @@ export const AddressSpaceListPage: React.FunctionComponent<AddressSpaceListPageP
     setSortValue({ index: index, direction: direction });
   };
 
+  const tableRows = addressSpacesList.map(row => getTableCells(row));
+
+  const onSelect = async (
+    event: React.MouseEvent,
+    isSelected: boolean,
+    rowIndex: number,
+    rowData: IRowData,
+    extraData: IExtraData
+  ) => {
+    let rows;
+    if (rowIndex === -1) {
+      rows = tableRows.map(a => {
+        const data = a;
+        data.selected = isSelected;
+        return data;
+      });
+      onSelectAllAddressSpace(
+        rows.map(row => row.originalData),
+        isSelected
+      );
+    } else {
+      rows = [...tableRows];
+      rows[rowIndex].selected = isSelected;
+      onSelectAddressSpace(rows[rowIndex].originalData, isSelected);
+    }
+  };
+
+  const actionResolver = (rowData: IRowData) => {
+    return getActionResolver(
+      rowData,
+      onChangeEdit,
+      onChangeDelete,
+      onDownloadCertificate
+    );
+  };
+
   return (
     <>
       {totalAddressSpaces > 0 ? (
-        <AddressSpaceList
```

Comment:
Any specific reason to remove the visual component with PF Table component?

Reply:
we don't need extra component for table only . table is used in AddressSpaceList component.